components/modal/modal.css

```diff
@@ -0,0 +1,26 @@
+.modal-container {
+  background-color: var(--light-color);
+  padding: 2rem;
+  margin: 2rem 0;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: 2rem;
+}
+.simple-modal {
+  padding: 1rem;
+  background-color: var(--white-color);
+  border: 2px solid var(--gray-color);
+  border-radius: 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+.action-btn {
```

Comment:
you can extract this to button component

Reply:
this class has been used for layouting